src/index.html

```diff
@@ -11,7 +11,184 @@
+      <fieldset enabled class="forma">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            class="field"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            class="field"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="number"
+            class="field"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+            class="field"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="Iacceptthetermoftheagreement"
+            class="field"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset enabled class="forma">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            class="field"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            class="field"
+            autocomplete="off"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset enabled class="forma">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              name="answer"
+              class="field"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label >
+            <input
+              type="radio"
+              name="answer"
+              class="field"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            class="field"
+          >
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            class="field"
+          >
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+
+          <select
+            name="brand"
+            multiple
+            class="field"
+            >
```

Comment:
please be attentive with spaces
```suggestion
          >
```